src/lib.rs

```diff
@@ -23,31 +23,33 @@ fn str_to_public_key(public_key: String) -> PublicKey {
 #[near_bindgen]
 impl LinkDrop {
     /// Allows given public key to claim sent balance.
-    pub fn send(&mut self, public_key: String) {
+    pub fn send(&mut self, public_key: String) -> PromiseOrValue<()> {
```

Comment:
Why not just `Promise`?

Reply:
Good point. Seems it works okay. I thought I tried it before, but it didn't work for me for some reason.